BITSWAP.md

```diff
@@ -1,200 +1,200 @@
 # ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) Bitswap
 
-**Authors(s)**:
+**Author(s)**:
+- Adin Schmahmann
 - David Dias
 - Jeromy Johnson
 - Juan Benet
 
-**Maintainers(s)**:
+**Maintainer(s)**:
```

Comment:
I'm not sure what this is supposed to be used for/if it really makes sense to keep around. Perhaps this is an interest group that I should be adding more people in the IPFS community who are interested in the Bitswap specs and improving them or being particularly involved in protocol upgrade proposals.